FusionIIIT/applications/online_cms/views.py

```diff
@@ -81,86 +89,86 @@ def course(request, course_code):
+        #         'type': 'video',
+        #         'channelId': 'channel_id'
+        #     }
+        #     videos_ids = []
+        #     r = requests.get(search_url, params=search_params)
+        #     # print(r)
+        #     results = r.json()['items']
+        #     for result in results:
+        #         videos_ids.append(result['id']['videoId'])
+
+        #     video_params = {
+        #         'key': settings.YOUTUBE_DATA_API_KEY,
+        #         'part': 'snippet,contentDetails',
+        #         'id': ','.join(videos_ids),
+        #         'maxResults': 9
+        #     }
+
+        #     p = requests.get(video_url, params=video_params)
+        #     results1 = p.json()['items']
+
+        #     for result in results1:
+        #         video_data = {
+        #             'id': result['id'],
+        #             # 'url': f'https://www.youtube.com/watch?v={result["id"]}',
+        #             'title': result['snippet']['title'],
+        #             # 'duration': int(parse_duration(result['contentDetails']['duration']).total_seconds() // 60),
+        #             # 'thumbnails': result['snippet']['thumbnails']['high']['url']
+        #         }
+
+        #         videos.append(video_data)
+        # else:
+        #     channel_url = "https://www.googleapis.com/youtube/v3/channels"
+        #     playlist_url = "https://www.googleapis.com/youtube/v3/playlistItems"
+        #     videos_url = "https://www.googleapis.com/youtube/v3/videos"
+
+        #     videos_list = []
+        #     channel_params = {
+        #         'part': 'contentDetails',
+        #         'id': 'channel_id',
+        #         'key': settings.YOUTUBE_DATA_API_KEY,
+        #     }
+        #     r = requests.get(channel_url, params=channel_params)
+        #     results = r.json()['items'][0]['contentDetails']['relatedPlaylists']['uploads']
+
+        #     playlist_params = {
+        #         'key': settings.YOUTUBE_DATA_API_KEY,
+        #         'part': 'snippet',
+        #         'playlistId': results,
+        #         'maxResults': 5,
+        #     }
+        #     p = requests.get(playlist_url, params=playlist_params)
+        #     results1 = p.json()['items']
+
+        #     for result in results1:
+        #         # print(results)
+        #         videos_list.append(result['snippet']['resourceId']['videoId'])
+
+        #     videos_params = {
+        #         'key': settings.YOUTUBE_DATA_API_KEY,
+        #         'part': 'snippet',
+        #         'id': ','.join(videos_list)
+        #     }
+
+        #     v = requests.get(videos_url, params=videos_params)
+        #     results2 = v.json()['items']
+        #     videos = []
+        #     for res in results2:
+        #         video_data = {
+        #             'id': res['id'],
+        #             'title': res['snippet']['title'],
+        #         }
+
+        #         videos.append(video_data)
```

Comment:
If you are commenting out the code and it is not needed in the future, kindly remove it.